clients/docs/includes/configs/cloud/java-sr.config

```diff
@@ -1,10 +1,14 @@
-# Kafka
+# Required connection configs for Kafka producer, consumer, and admin
 bootstrap.servers={{ BROKER_ENDPOINT }}
 security.protocol=SASL_SSL
 sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='{{ CLUSTER_API_KEY }}' password='{{ CLUSTER_API_SECRET }}';
 sasl.mechanism=PLAIN
+client.dns.lookup=use_all_dns_ips
```

Comment:
```suggestion

# Best Practice for all Java clients
client.dns.lookup=use_all_dns_ips
```

Reply:
Is this really worth it? I think most users won't care about the distinction. I'd just leave it as part of the previous block and remove it in a few months once 2.6+ is a common choice for new apps.

Reply:
> Is this really worth it? I think most users won't care about the distinction. I'd just leave it as part of the previous block and remove it in a few months once 2.6+ is a common choice for new apps.

Yes, IMO the distinction is important, see (a) from earlier comment:

```
(a) differentiating what is mandatory to connect to Confluent Cloud from what gives better experience
(b) differentiating which configs are for all clients versus just producer versus just consumer, etc
(c) portability of properties file -- users may take the properties intended for CCloud and then blindly reuse the same configuration file for testing on a local cluster.
```

Reply:
The question is why is this important in this case. This config is necessary for correctness, it's the new default, it's safe to use in Cloud and CP, it's safe to use in every client, there is no downside to using it and it's a connection related config. I don't think we're really helping people by separating it (unlike the acks config). If you really insist, I'll make the change you suggested, but I strongly believe it makes it worse rather than better.

Reply:
@ijuma 

1. If `client.dns.lookup=use_all_dns_ips` is going to be added into these files, then I strongly lean towards having it commented as suggested.   (Or we leave it out since it's the new default?)

2. wrt "it's safe to use in every client" -- this PR currently modifies only the Java files.  If it's applicable to use in every client, then for consistency this PR should modify the other files as well (see all files within subdirectory https://github.com/confluentinc/examples/tree/6.0.0-post/clients/docs/includes/configs)

Reply:
I mean in every Java client. It's not needed by other clients. The main reason why I'm adding it is that not having this causes outages in production. It's not really "best practice", it's required.

Reply:
If the concern is to prevent/avoid outages in CCloud production, then the target user is those running _older_ code, which means they are likely not referring to these pages anyway.  Is it possible that the users who would refer to these pages to get started are likely running AK 2.6 already?